next.config.js

```diff
@@ -1,3 +1,11 @@
 module.exports = {
   reactStrictMode: true,
+  webpack: (configuration) => {
+    configuration.module.rules.push({
+      test: /\.md$/,
+      use: "frontmatter-markdown-loader",
+      options: { mode: ["react-component"] },
```

Comment:
incorrect webpack loaders `options` usage

```suggestion
      use: {
        loader: "frontmatter-markdown-loader",
        options: { mode: ["react-component"] },
      }
```